tests/e2e/monitoring/test_monitoring_after_reboot_master_node.py

```diff
@@ -0,0 +1,99 @@
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import tier4, E2ETest, ignore_leftovers
+from ocs_ci.utility import aws
+from tests.sanity_helpers import Sanity
+from ocs_ci.ocs.monitoring import check_pvcdata_collected_on_prometheus
+from ocs_ci.ocs.node import wait_for_nodes_status, get_typed_nodes
+from ocs_ci.utility.retry import retry
+from ocs_ci.ocs.exceptions import CommandFailed
+
+
+logger = logging.getLogger(__name__)
+
+
+@retry(CommandFailed, tries=10, delay=3, backoff=1)
+def wait_for_master_node_to_be_running_state():
+    """
+    Waits for the all the nodes to be in running state
+    """
+    wait_for_nodes_status()
+
+
+@pytest.fixture()
+def test_fixture(pod_factory, num_of_pod=2):
+    """
+    Setup and teardown
+    """
+    pod_objs = [
+        pod_factory(interface=constants.CEPHBLOCKPOOL,
+                    status=constants.STATUS_RUNNING
+                    ) for _ in range(num_of_pod)
+    ]
+
+    # Check for the created pvc metrics on prometheus pod
+    for pod_obj in pod_objs:
+        assert check_pvcdata_collected_on_prometheus(pod_obj.pvc.name), (
+            f"On prometheus pod for created pvc {pod_obj.pvc.name} related data is not collected"
+        )
+
+    return pod_objs
+
+
+@pytest.mark.polarion_id("OCS-709")
+@ignore_leftovers
```

Comment:
why?

Reply:
When master node is rebooting 'oauth-openshift-xxxxxxxxxx-xxxxx' pod  is re-spinning and coming with new name. So it fails during environment check saying one resource added and one resource removed. 
https://paste.fedoraproject.org/paste/MKy2rYPKpLWd~E2O4tkJDA

Reply:
ack